.buildkite/validate_swift_package

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -eu
+#
+# Any arguments passed to this script will be passed through to `fastlane`.
+# If no argument is passed, the `test` lane will be called by default.
+
+echo "--- :rubygems: Setting up Gems"
+install_gems
+
+echo "--- :test_tube: Building and testing the Swift Package"
+
+# For some reason this fixes a failure in `lib lint`
+# https://github.com/Automattic/buildkite-ci/issues/7
+xcrun simctl list >> /dev/null
```

Comment:
If that is supposed to solve an issue with `[pod] lib lint`, then is it still necessary for SwiftPM (as opposed to for CocoaPods)?

PS: I know the explanation for the "for some reason": this is because on a machine where Xcode has been newly installed and never launched, and on which `xcrun simctl` hasn't been run on either, the list of simulators is completely empty. It only gets populated by the default list of simulators shipped with that version of Xcode after you launch Xcode at least once, or run `xcrun simctl list` at least once.

(See [this code comment](https://github.com/wordpress-mobile/WordPress-iOS/blob/c4a83561d8794b22838f4baaa881a11316d6fb98/fastlane/lanes/screenshots.rb#L222-L224) for a case where I encountered a similar issue)

Reply:
🤦‍♂️ I forgot to delete this file. But this line has been removed in [the buildkite plugin PR](https://github.com/Automattic/bash-cache-buildkite-plugin/pull/33/files#diff-47e1c0a23dba0133fe98f93548c51465ab5a62a678e317cef3e13539a666a9d8)